src/components/SigningModal/TransactionStepper/TransactionStepper.js

```diff
@@ -236,6 +290,11 @@ TransactionStepper.propTypes = {
       }),
     })
   ).isRequired,
+  info: PropTypes.shape({
```

Comment:
Could we find a better name for this, especially since it's a mapping? Perhaps `infoDescriptions`, to mirror `steps.descriptions?`